cypress/e2e/undo-redo/block-configs-should-update-flow-state/content--blocks-configs.cy.js

```diff
@@ -0,0 +1,90 @@
+describe('mutate flow state for CONTENT blocks', () => {
+  beforeEach(() => {
+    cy.createFlow({
+      label: 'adding blocks test',
+    })
+  })
+
+  it('should validate Location Response Block', () => {
+    const blockConfigs = {
+      accuracy: {
+        threshold: 25,
+        timeout: 134
+      },
+    }
+    const getStore = () => cy.window().its('store')
+    const flowModuleState = () => getStore().its('state.flow')
+    const flowsListState = () => flowModuleState().its('flows')
+
+    cy.addBlock(['Location Response'])
+    const firstBlockState = () => flowsListState().its('[0].blocks[0]')
+
+    cy.get('[data-cy="accuracy-threshold--editor"]')
+      .find('input')
```

Comment:
@pascaleli ,  it's important to find the writable element (eg: input, textarea) before you call `type()`. Otherwise, you won't be able to type something